ProgramLauncher.c

```
@@ -0,0 +1,391 @@
+typedef struct ProgramFileRef_ {
+   dev_t dev;
+   ino_t ino;
+   char* path;
+} ProgramFileRef;
+
+extern char** environ;
+
+static gid_t ProgramLauncher_savedSetGid = (gid_t)-1;
+static uid_t ProgramLauncher_savedSetUid = (uid_t)-1;
+
+void ProgramLauncher_done(ProgramLauncher* this) {
```

Cppcheck (2.20.0)

[style] The function 'ProgramLauncher_done' is never used.

(unusedFunction)